roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,3 @@
 dependencies:
   - pip
+  - logrotate
```

Comment:
If we have the log rotation data as a role default, you can then do

```
dependencies:
  - role: pip
  - role: logrotate
    logrotate_configs: "{{ zuul.logrotate_configs }}"
```

This way the info about the logging stays with the service, and is only altered if some site REALLY needs to via inventory vars. When it's all in inventory it becomes tedious to create new inventories.